src/index.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable */
+import { getOptions, isUrlRequest, stringifyRequest } from 'loader-utils';
+import validateOptions from 'schema-utils';
+
+import resolver from './resolver';
+import processor from './processor';
+
+import SyntaxError from './Error';
+
+/**
+ * CSS Loader
+ *
+ * > Loads CSS as Modules for webpack
+ *
+ * @author Tobias Koppers (@sokra)
+ *
+ * @version 1.0.0
+ * @license MIT
+ *
+ * @requires 'loader-utils'
+ * @requires 'schema-utils'
+ *
+ * @requires './resolver'
+ * @requires './processor'
+ *
+ * @method loader
+ *
+ * @param  {String} src  Source
+ * @param  {Object} map  Sourcemap
+ * @param  {Object} meta Metadata
+ *
+ * @return {cb} cb       Callback
+ */
+export default function loader (src, map, meta) {
```

Comment:
Yep this should work according to sokra, needs to be tested though :), the third argument of the callback can take a meta `{Object}`

```js
function loader (src, map, meta) {// {String} | {Object} | {Object}
   ....
   .then(({ css, map, messages }) => cb(null, css, map, { messages })
}